examples/windows/kubernetes.json

```diff
@@ -3,7 +3,7 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorRelease": "1.13"
+      "orchestratorVersion": "1.14.1"

```

Comment:
let's not pin to a specific k8s version, this is hard to maintain overtime as it will need to be updated everytime we introduce a new patch. Why is this change relevant to this PR?

Reply:
This was the only way I could get it to work with the Windows setup. I kept getting errors like "version x does not support windows nodes, use one of these versions instead: ....."

Reply:
I changed it to `orchestratorRelease: 1.14`, that seems to work